sasmodels/sasview_model.py

```diff
@@ -677,7 +694,7 @@ def calculate_Iq(self, qx, qy=None):
         with calculation_lock:
             return self._calculate_Iq(qx, qy)
 
-    def _calculate_Iq(self, qx, qy=None):
+    def _calculate_Iq(self, qx, qy=None, Fq=False, effective_radius_type=1):
```

Comment:
This method returns two results now. The callers use either 'result' or 'lazy_results', disregarding the other. This is wasteful. It would be preferable to have two separate _calculate_Iq methods returning 'result' and 'lazy_results' as the only thing.

Reply:
The interface between sasview and sasmodels needs to be completely replaced, with sasview accessing the sasmodels ModelInfo directly (ticket #806).  Let's leave the current ball of glue and sticky tape in place for now.